packages/react/src/components/accordion/accordion-item.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Theme } from '../../themes';
+import { Button } from '../buttons/button';
+import { Icon } from '../icon/icon';
+import { Heading, Type, Tag } from '../heading/heading';
+
+interface AccordionItemProps {
+    title: string;
+    id?: string;
+    type?: Type | undefined;
+    tag?: Tag | undefined;
+    expanded?: boolean | undefined;
+    disabled?: boolean | undefined;
+    onToggle?: () => void;
+    onKeyDown?: ((event: React.KeyboardEvent<HTMLButtonElement>) => void) | undefined;
+    children: React.ReactNode;
+    buttonRef?: React.RefObject<HTMLButtonElement> | undefined;
+}
+
+const AccordionSection = styled.section<{ theme: Theme }>`
+    background: ${({ theme }) => theme.greys['colored-white']};
+    border-color: ${({ theme }) => theme.greys.grey};
+    border-radius: 0 0 var(--border-radius-2x) var(--border-radius-2x);
+    border-style: solid;
+    border-width: 0;
+    margin-bottom: var(--spacing-1x);
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.5s ease, border-width 0.5s ease;
+    &.expanded {
+        border-width: 1px;
```

Comment:
`border-width: 0 1px 1px 1px;` so you wouldn't need the `border-top-width: 0;` on the next line.

Reply:
done